server/routes/staffDashboard/staffDashboardController.test.ts

```diff
@@ -0,0 +1,96 @@
+import request from 'supertest'
+import type { Express } from 'express'
+import RpService from '../../services/rpService'
+import Config from '../../s3Config'
+import { stubPrisonerDetails } from '../testutils/testUtils'
+import { configHelper } from '../configHelperTest'
+import { appWithAllRoutes } from '../testutils/appSetup'
+
+let app: Express
+let rpService: jest.Mocked<RpService>
+const config: jest.Mocked<Config> = new Config() as jest.Mocked<Config>
+
+beforeEach(() => {
+  rpService = new RpService() as jest.Mocked<RpService>
+  jest.mock('applicationinsights', () => jest.fn())
+  configHelper(config)
+
+  app = appWithAllRoutes({
+    services: {
+      rpService,
+    },
+  })
+  stubPrisonerDetails(rpService)
+})
+afterEach(() => {
+  jest.resetAllMocks()
+})
+describe('getView', () => {
+  it('Happy path with default query params', async () => {
+    await request(app)
```

Comment:
Might be some mocking missing hear for the call to the RP Service?

Reply:
done